kuma/payments/jinja2/payments/includes/payments-form.html

```diff
@@ -90,6 +90,7 @@
         donationChoices: {
             oneTime: {{payments_form_donation_choices_json}},
             recurring: {{recurring_payment_form_donation_choices_json}}
-        }
+        },
+        isAuthenticated: '{{user.is_authenticated()}}' === 'True'
```

Comment:
ahh ok, locally when developing `'{{user.is_authenticated}}'` rendered as `'CallableBool(True)'`.
Calling it forced it to be `True`. So would this work in staging/prod? 

Alternatively I could catch this by doing `{% if user.is_authenticated %}true{% else %}false{% endif %}` would that be preferred?

Reply:
I think either ``user.is_authenticated()`` or ``user.is_authenticated`` is OK for today. It is on my mind because of https://github.com/mozilla/kuma/pull/5089, and can be fixed later.

I prefer your section version, ``{% if user.is_authenticated %}true{% else %}false{% endif %}``, it looks less like we're forcing Python into JavaScript.